tests/interpreter_integration.rs

```diff
@@ -18,10 +18,48 @@
 
 #![cfg(test)]
 use pact::{
-    interpreter::{self, InterpErr, OpCode},
+    interpreter::{self, InterpErr},
+    interpreter::{Comparator, OpCode, OpComp, OpConj, OpIndices, OpInvert, OpLoad, OpType},
     types::{Numeric, PactType, StringLike},
 };
 
+// Helper Macros
+// Create a comparator op code
+macro_rules! comparator {
```

Comment:
nit: not sure we need macro vs. some nice constructors

Reply:
Nice! Done 😄